resources/dicts/patrols/general/training.json

```diff
@@ -1094,4 +1094,45 @@
     "antagonize_text": null,
     "antagonize_fail_text": null
 }
-]
+{
+    "patrol_id": "gen_training_twoapps_forbiddentunnels1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["two_apprentices", "rel_two_apps", "big_change", "trust",  "scar", "injury", "broken back", "app_only", “training”],
```

Comment:
if you have the "scar" tag then you need to also include the name of the scar that this patrol should be giving on classic mode.